backends-common/elasticsearch/src/main/java/org/apache/james/backends/es/ElasticSearchConfiguration.java

```diff
@@ -78,6 +79,12 @@ public Builder nbReplica(int nbReplica) {
             return this;
         }
 
+        public Builder waitForActiveShards(int waitForActiveShards) {
+            Preconditions.checkArgument(waitForActiveShards >= 0, "You need the number of waitForActiveShards to be positive");
```

Comment:
should it be `waitForActiveShards > 0` ?

Reply:
The only forbidden values in ES are the negatives.

Reply:
then you should change the message from `to be positive` to `not to be negative`